### PR TITLE
RIOT port for the KW01Z128 SiP  and Phytec PhyWave-KW01 Evaluations-Board

### PR DESCRIPTION
**Pull request by jfischer-phytec-iot**
This PR adds support for Freescale KW01Z128 SiP (Cortex-M0 + Sub-1 GHz Radio) and Phytec PhyWave-KW01 Evaluations-Board.
~~This PR depends on #2265.~~

### CODE REVIEW

**Comment by thomaseichinger** [comment]
As #2265 got merged, is this PR ready for review?

**Comment by jfischer-phytec-iot** [comment]
It needs update and more testing. It will take 2-3 weeks.

**Comment by jfischer-phytec-iot** [comment]
@thomaseichinger ping :checkered_flag:

**Comment by jfischer-phytec-iot** [comment]
outdated, not ready for review

**Comment by PeterKietzmann** [comment]
Wow, that was a fast reaction :-) !

**Comment by PeterKietzmann** [comment]
On March 20th we already had the ":checkered_flag:". Any news here?

File: boards/pba-d-01-kw01/Makefile.features [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_cpuid
```
Comment (gebart) [inline boards/pba-d-01-kw01/Makefile.features]: Please add `FEATURES_MCU_GROUP = cortex_m0`

**Comment by OlegHahm** [comment]
@jfischer-phytec-iot, ping?

**Comment by OlegHahm** [comment]
Any news?

**Comment by OlegHahm** [comment]
Still WIP and no update for half a year now - highly unlikely to have this ready by next Wednesday. @jfischer-phytec-iot, do you still plan to continue the work on this?

**Comment by jfischer-phytec-iot** [comment]
I close it for now, I will open it again when the time comes.

**Pull request closed without merging**